toolchest_client/tools/api.py

```diff
@@ -135,6 +135,72 @@ def kraken2(output_path, inputs=[], database_name="standard", database_version="
     instance.run()
 
 
+def shogun_align(inputs, output_path, database_name="shogun_standard", database_version="1", tool_args=""):
+    """Runs Shogun (for alignment) via Toolchest.
+
+    :param tool_args: (optional) Additional arguments to be passed to Shogun.
+    :param database_name: (optional) Name of database to use for Shogun alignment. Defaults to the pre-built DB files at https://github.com/knights-lab/SHOGUN.
+    :param database_version: (optional) Version of database to use for Bowtie 2 alignment.
+    :type database_version: str
+    :param inputs: Path to be passed in as input.
+    :param output_path: Path (client-side) where the output file will be downloaded.
+
+    Usage::
+
+        >>> import toolchest_client as toolchest
+        >>> toolchest.shogun_align(
+        ...     database_name="DB_name",
+        ...     database_version="version_number",
+        ...     inputs="./path/to/input",
+        ...     output_path="./path/to/output",
+        ... )
+
+    """
+
+    instance = ShogunAlign(
+        tool_args=tool_args,
+        output_name='output.txt',  # TODO: add actual outputs
+        inputs=inputs,
+        output_path=output_path,
+        database_name=database_name,
+        database_version=database_version
+    )
+    instance.run()
+
+
+def shogun_filter(inputs, output_path, database_name="shogun_standard", database_version="1", tool_args=""):
+    """Runs Shogun (for filtering human genome content) via Toolchest.
+
+    :param tool_args: (optional) Additional arguments to be passed to Shogun.
+    :param database_name: (optional) Name of database to use for Shogun alignment. Defaults to the pre-built DB files at https://github.com/knights-lab/SHOGUN.
+    :param database_version: (optional) Version of database to use for Bowtie 2 alignment.
+    :type database_version: str
+    :param inputs: Path to be passed in as input.
+    :param output_path: Path (client-side) where the output file will be downloaded.
+
+    Usage::
+
+        >>> import toolchest_client as toolchest
+        >>> toolchest.shogun_filter(
+        ...     database_name="DB_name",
+        ...     database_version="version_number",
+        ...     inputs="./path/to/input",
+        ...     output_path="./path/to/output",
+        ... )
+
+    """
+
+    instance = ShogunFilter(
+        tool_args=tool_args,
+    output_name='output.txt',  # TODO: add actual outputs
+        inputs=inputs,
+        output_path=output_path,
+        database_name=database_name,
+        database_version=database_version
```

Comment:
LGTM, is this added in the staging db yet?

Reply:
Yes :+1: